gax-java/gax/clirr-ignored-differences.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- see http://www.mojohaus.org/clirr-maven-plugin/examples/ignored-differences.html -->
+<differences>
+  <difference>
+    <!-- add default stream methods to `Page` interface -->
+    <differenceType>7012</differenceType>
+    <className>com/google/api/gax/paging/Page</className>
+    <method>* stream*(*)</method>
```

Comment:
It's strange that CLIRR is complaining the change. Adding default method in interface shouldn't cause a breaking changes.

Reply:
@JoeWang1127 Just in case, would you build this gax-java locally (with -SNAPSHOT version) and try to use in java-storage to see anything breaks? You ran some storage samples that you used to understand the request.

Reply:
In [Specifying ignored differences](https://www.mojohaus.org/clirr-maven-plugin/examples/ignored-differences.html) of Clirr Maven Plugin doc, 7012 has the following definition:
```
7012 (Method Added to Interface): className, method
```
So adding a method to interface, default or not, is a breaking change to this rule.

Also, there's 7013:
```
7013 (Abstract Method Added to Class): className, method
```
which is not complain.

Reply:
Clirr isn't aware of java 8 default methods. It is looking at the bytecode from the perspective of java7 binary compatibility. When adding a default method it is safe to specify an ignore rule.

Reply:
@BenWhitehead Thanks for the clarification.